minisom.py

```diff
@@ -390,15 +391,23 @@ def activation_response(self, data):
         for x in data:
             a[self.winner(x)] += 1
         return a
+    
+    def dist_mat(self, data):
```

Comment:
this function should be called `_distance_from_weights` and should have a unit test.